### ISSUE TITLE
zotero not working with latest firefox version

### ISSUE THREAD

**Issue by nico202** [NixOS/nixpkgs#18055]
## Issue description

Zotero does not start on NixOS unstable

### Steps to reproduce

* check you have both firefox and zotero updated (like `nixos-rebuild switch --upgrade`)
* launch zotero: `zotero`

```
TypeError: Proxy.create is not a function
chrome://zotero/content/xpcom/translation/translate_firefox.js:163
Zotero.Translate.DOMWrapper<@chrome://zotero/content/xpcom/translation/translate_firefox.js:163:23
@chrome://zotero/content/xpcom/translation/translate_firefox.js:40:31
makeZoteroContext@file:///nix/store/s4063w30mcp39kkj7ll71qpykrqkh7bs-zotero-4.0.29/libexec/zotero/components/zotero-service.js:255:4
ZoteroService@file:///nix/store/s4063w30mcp39kkj7ll71qpykrqkh7bs-zotero-4.0.29/libexec/zotero/components/zotero-service.js:312:4
XPCOMUtils__getFactory/factory.createInstance@resource://gre/modules/XPCOMUtils.jsm:383:19
loadZotero@resource://gre/modules/addons/XPIProvider.jsm -> file:///home/nixo/.zotero/zotero/jofw9o0d.default/extensions/zotxt@e6h.org/bootstrap.js:47:18
loadEndpoints@resource://gre/modules/addons/XPIProvider.jsm -> file:///home/nixo/.zotero/zotero/jofw9o0d.default/extensions/zotxt@e6h.org/bootstrap.js:579:5
observer.observe@resource://gre/modules/addons/XPIProvider.jsm -> file:///home/nixo/.zotero/zotero/jofw9o0d.default/extensions/zotxt@e6h.org/bootstrap.js:591:9
```

## Technical details

Zotero on nix uses sytem firefox (see here: [zotero-nix](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/office/zotero/zotero.sh))
while using it is not supported (same problem here on [zotero forum](https://forums.zotero.org/discussion/61703/zotero-standalone-doesnt-start/))

* System: 16.09pre89269.8512747 (Flounder)
* Nix version: nix-env (Nix) 1.11.2
* Nixpkgs version: "16.09pre89269.8512747"
* Firefox version: Mozilla Firefox 48.0.1

**Comment by nico202** [comment]
Upgrading to release 4.0.29.5 does not solve the problem (that is fixed in the xpi plugin https://github.com/zotero/zotero/commit/b4ac43fd7758304cc71c9626cf0576aee284b78a)

**Comment by vcunat** [comment]
Maybe the standalone zotero would be easier to maintain if we kept it using the ESR firefox.

**Comment by ttuegel** [comment]
> Maybe the standalone zotero would be easier to maintain if we kept it using the ESR firefox.

It used to be pinned to its own Firefox build; what happened?

**Comment by edwtjo** [comment]
@ttuegel I've changed to using esr in 0edc7ee as it solves this annoying issue, hope you don't mind.

**Comment by vcunat** [comment]
@edwtjo: that doesn't work for me
```
Error: Platform version '45.3.0' is not compatible with
minVersion >= 38.0
maxVersion <= 45.3.0esr
```

**Comment by edwtjo** [comment]
@vcunat sorry about that, I must have copied from the wrong terminal :/ I've addressed this in #18297
